### PR TITLE
[#44825] Add .env to Environment Configuration Docs

### PR DESCRIPTION
**Pull request by ghost**
[#44825] Add .env to Environment Configuration Docs

https://community.openproject.org/work_packages/44825

### CODE REVIEW

File: docs/installation-and-operations/configuration/README.md [hunk 1]
```diff
@@ -8,19 +8,155 @@ sidebar_navigation:
...
+# Packaged installation
 
-## List of options
+The file `/opt/openproject/.env.example` contains some information to learn more. The file `/opt/openproject/conf.d/env` is used for parsing the variables and your custom values to your configuration.
+
+To configure the environment variables such as the number of web server threads OPENPROJECT_HTTPS, copy the `.env.example` to `/etc/openproject/conf.d/env` and add the environment variables you want to configure. The variables will be automatically loaded to the application’s environment.
+
+After changing the file `/etc/openproject/conf.d/env`  the command `sudo openproject configure` must be issued
+
+If you would like to change a value of a variable without a downtime of OpenProject you are able to configure the environment variable by using the command:
```
Comment (oliverguenther) [inline docs/installation-and-operations/configuration/README.md]: This is not correct. Using `openproject config:set ...` has the same effect as putting it to `/etc/openproject/conf.d/other` and still requires a restart of the server

Comment (ghost) [reply docs/installation-and-operations/configuration/README.md]: thank you for clarifying this Oliver I will change the text

Comment (ghost) [reply docs/installation-and-operations/configuration/README.md]: @oliverguenther i changed your remark

**Comment by ghost** [comment]
@oliverguenther I continued in editing the configuration files, this PR will change nearly everything to be configured in via ENV, just one thing i will not be able to do is the repository page, so i changed that to being outdated in a NOTE on the first row

**Review by oliverguenther — APPROVED** [review]

**Pull request merged**